src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -475,7 +454,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="ComputeFilesToPublish"
           DependsOnTargets="ComputeResolvedFilesToPublishList;
                             ILLink;
-                            CreateReadyToRunImages">
+                            CreateReadyToRunImages;
+                            GeneratePublishDependencyFile;
+                            BundlePublishDirectory">
```

Comment:
Since `BundlePublishDirectory` runs before files are copied to the publish directory now, should it be renamed?

Reply:
Thanks @dsplaisted. I've renamed the target `BundlePublishDirectory` to `GenerateSingleFileBundle`.